### PR TITLE
fix: hlb cifar types

### PR DESCRIPTION
**Pull request by seendsouza**
```
❯ mypy examples/hlb_cifar10.py
Success: no issues found in 1 source file
```

### CODE REVIEW

**Comment by g1y5x3** [comment]
It would be nice if CI can spot this so they can be fixed on the fly next time.

**Comment by seendsouza** [comment]
Hmm I think it's because it's in examples. There are 228 errors in other examples right now @g1y5x3

File: examples/hlb_cifar10.py [hunk 1]
```diff
@@ -127,7 +129,8 @@ def _cov(X):
     def _patches(data, patch_size=(kernel_size,kernel_size)):
       h, w = patch_size
       c = data.shape[1]
-      return np.lib.stride_tricks.sliding_window_view(data, window_shape=(h,w), axis=(2,3)).transpose((0,3,2,1,4,5)).reshape((-1,c,h,w))
+      axis: SupportsIndex = (2, 3) # type: ignore
```
Comment (geohot) [inline examples/hlb_cifar10.py:132]: Hmm, why? Does the array [2,3] does work?

Comment (seendsouza) [reply examples/hlb_cifar10.py:132]: Nope. It's very weird. Int works, but not tuple or list

**Comment by geohot** [comment]
`extra` I don't care about, but `examples`, `models`, and `test` should be type checked. However, this needs to be done tastefully, not with a huge commit, but with thought into what should just be deleted.

**Pull request merged**